ui/src/logs/components/LogsTable.tsx

```diff
@@ -181,22 +214,39 @@ class LogsTable extends PureComponent<Props, State> {
         value = _.replace(value, '\\n', '')
         break
       case 'severity':
-        return (
-          <div style={style} key={key}>
-            <div
-              className={`logs-viewer--dot ${value}-severity`}
-              title={this.severityLevel(value)}
-            />
-          </div>
+        value = (
+          <div
+            className={`logs-viewer--dot ${value}-severity`}
+            title={this.severityLevel(value)}
+          />
         )
     }
 
+    let backgroundColor = ''
+    if (rowIndex === this.state.currentRow && columnIndex > 0) {
+      backgroundColor = HIGHLIGHT_COLOR
+    }
+
     return (
-      <div style={style} key={key}>
+      <div
+        style={{...style, padding: '5px', backgroundColor}}
+        key={key}
+        onMouseOver={this.handleMouseOver}
+        data-index={rowIndex}
+      >
         {value}
       </div>
     )
   }
+
+  private handleMouseOver = (e: MouseEvent<HTMLElement>) => {
+    const target = e.target as HTMLElement
```

Comment:
is there a reason you need to cast this as HTMLElement?

Reply:
yeah, I had an issue with accessing the dataset that was fixed by casting. Going to revisit on the next chunk of work for this feature set.